packages/get-platform/src/__tests__/getPlatform.test.ts

```
@@ -1,20 +1,38 @@
-import { jestConsoleContext, jestContext } from '../test-utils'
 
-const ctx = jestContext.new().add(jestConsoleContext()).assemble()
+const it = test.extend<{
+  consoleMock: {
+    log: MockInstance
+    warn: MockInstance
+    error: MockInstance
+  }
+}>({
+  // eslint-disable-next-line no-empty-pattern
+  consoleMock: async ({}, use) => {
```

Biome (2.4.4)

[error] Unexpected empty object pattern.

(lint/correctness/noEmptyPattern)